set.mm

```diff
@@ -84691,16 +84691,20 @@ of Infinity (see comments in ~ fin2inf ).  (Contributed by NM,
 $)
 
   $c fi $.
-  $( Extend class notation with the function whose value is the class of all
-     the finite intersections of the elements of a given set. $)
+  $( Extend class notation with the function whose value is the class of finite
+     intersections of the elements of a given set. $)
   cfi $a class fi $.
 
   ${
     $d x y z $.
-    $( Function whose value is the class of all the finite intersections of the
-       elements of ` x ` .  (Contributed by FL, 27-Apr-2008.) $)
-    df-fi $a |- fi = ( x e. _V |-> { z |
-                  E. y e. ( ~P x i^i Fin ) z = |^| y } ) $.
+    $( Function whose value is the class of finite intersections of the
+       elements of the argument.  Note that the empty intersection being the
+       universal class, hence a proper class, it cannot be an element of that
```

Comment:
typo: ...class, cannot...

Reply:
I don't see a typo: I would repeat the subject since there is no comma:
> Jack being tall, he is not short.
> Jack, being tall, is not short.

We may have to ask a native speaker, e.g. @jkingdon